File: Placements.DataAccess/placement/Models/JobpostingSelectedstudent.cs
﻿using System;
using System.Collections.Generic;

namespace Placements.DataAccess.Placement.Models;

public partial class JobpostingSelectedstudent
{
    public long Id { get; set; }

    public long? JobPostingId { get; set; }

    public long? StudentId { get; set; }

    public ulong? HasAcceptedOffer { get; set; }

    public DateTime? DateOfJoining { get; set; }

    public virtual Jobposting? JobPosting { get; set; }

    public virtual Tblstudent? Student { get; set; }
}

File: web-app/src/app/services/types/JobpostingSelectedstudent.ts
import { Jobposting } from "./Jobposting";
import { StudentSkill } from "./StudentSkill";
import { Tblstudent } from "./Tblstudent";

export type JobpostingSelectedstudent = {
  industriesWithCompanyId: any;
  Id: number;
  JobPostingId: number;
  StudentId: number;
  HasAcceptedOffer: number;
  DateOfJoining: Date;
  OfferLetterSentDate: Date;
  OfferLetterExpiryDate: Date;
  JobPosting: Jobposting;
  Student: Tblstudent;
  StudentSkills: StudentSkill[];
};
